### DISCUSSION TITLE
Migration to Astro 2.0

### DISCUSSION THREAD

**Discussion by prototypa** [arthelokyo/astrowind#103] [Announcements]
**AstroWind** has been migrated to `Astro 2.0` and we have new features.

You can find the version with `Astro 1.0` in the [**0.x branch**](https://github.com/onwidget/astrowind/tree/0.x).
We are working on releasing **AstroWind** version 1.0 before the end of the month.

**Comment by glenlong** [comment]
Hey. Just curious - any update on the timetable for AstroWind 1.0?
